preview/MsixCore/msixmgr/msixmgr.rc

```diff
@@ -123,21 +167,3 @@ BEGIN
 /////////////////////////////////////////////////////////////////////////////
 #endif    // not APSTUDIO_INVOKED
 
-/////////////////////////////////////////////////////////////////////////////
-//
-// Version
-//
-#define VER_FILEVERSION 0,0,0,0
-#define VER_FILEVERSION_STR "0.0.0.0"

```

Comment:
keep these variables

Reply:
I did not change these at all. I think its something because of my compiler.

Reply:
These still exist, the order for these got switched to a different line because of adding a new string.